<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="ITSD-NB-029754" failures="1" tests="4" name="com.GSP.testobjects.GSPRegressionSuiteTest" time="28.904" errors="0" timestamp="16 May 2023 07:13:36 GMT" skipped="0">
  <testcase classname="com.GSP.testobjects.GSPRegressionSuiteTest" name="successfulLogin" time="4.489"/>
  <testcase classname="com.GSP.testobjects.GSPRegressionSuiteTest" name="logOutGSP" time="5.478"/>
  <testcase classname="com.GSP.testobjects.GSPRegressionSuiteTest" name="cancelMRDRequest" time="12.916">
    <failure message="expected [pass] but found [fail]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [pass] but found [fail]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:178)
	at com.GSP.testobjects.GSPRegressionSuiteTest.cancelMRDRequest(GSPRegressionSuiteTest.java:1155)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:577)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:414)
	at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
]]>
    </failure>
  </testcase> <!-- cancelMRDRequest -->
  <testcase classname="com.GSP.testobjects.GSPRegressionSuiteTest" name="invalidPasswordLogin" time="6.021"/>
</testsuite> <!-- com.GSP.testobjects.GSPRegressionSuiteTest -->
